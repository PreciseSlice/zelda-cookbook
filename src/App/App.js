import React, { Component } from 'react';
// import { Route } from 'react-router-dom';
import './App.css';

class App extends Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <div id="small-zelda-header"></div>
        </header>
        <nav></nav>
      </div>
    )
  }
}

export default App;
