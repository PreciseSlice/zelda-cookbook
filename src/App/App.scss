.App-header {
  background: linear-gradient(to right, #1C3744 20%, #ddd0b6);
  height: 150px;
  width: 100vw;
  padding: 20px;
  z-index: -999;
}

#small-zelda-header {
  background-image: url('../assets/small-zelda-header.png');
  background-size: contain;
  background-repeat: no-repeat;
  height: 100%;
  width: 100%;
}

body {
  background-image: url('../assets/floral-old-paper-or-parchment.jpg');
  background-size: cover;
  background-repeat: no-repeat;
  height: 100vh;
  width: 100vw;
  opacity: 0.75;
}
