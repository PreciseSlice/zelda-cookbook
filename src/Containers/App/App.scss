@import url("https://fonts.googleapis.com/css?family=PT+Serif:700");

* {
  box-sizing: border-box;
}

.App {
  display: grid;
  grid-template-columns: 300px auto;
}

.sidebar {
  background: linear-gradient(to bottom, #1C3744 20%, #ddd0b6);
  height: 100vh;
  padding: 20px;
}

#small-zelda-header {
  background-image: url('../../assets/small-zelda-header.png');
  background-size: contain;
  background-repeat: no-repeat;
  box-shadow: 0 0 20px 10px #1C3744;
  height: 194px;
  width: 253px;
}

#search-container {
  align-items: stretch;
  display: grid;
  justify-content: space-around;
  grid-template-columns: 80% auto;
  grid-template-rows: 1fr;
}

#dropDown {
  grid-row-start: 2;
  justify-self: stretch;
}

body {
  background-image: url('../../assets/floral-old-paper-or-parchment.jpg');
  background-size: cover;
  background-repeat: no-repeat;
  height: 100vh;
  width: 100vw;
  opacity: 0.75;
}
