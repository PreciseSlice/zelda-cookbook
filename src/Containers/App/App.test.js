/* eslint-disable */
import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<App />);

    expect(wrapper).toMatchSnapshot();
  });
})

// describe('componentDidMount', () => {
//     it('should call getIngredients', () => {
//       expect(apiCalls.getIngredients).toHaveBeenCalled();
//     });

//     it('should call getIngredientNames with an array of raw ingredients', () => {
//       const mockRawIngredients = [{
//         id: 1,
//         category: 'food',
//         duration: '0:30',
//         effect: null,
//         hearts: '0.5',
//         name: 'Apple',
//         resale: '3',
//         type: 'Fruit',
//         created_at: '2018-04-07T02:42:46.591Z',
//         updated_at: '2018-04-07T02:42:46.591Z'
//       }]
//       expect(dataCleaner.getIngredientNames).toHaveBeenCalledWith(mockRawIngredients)
//     });

//     it('should setState of ingredients to be an array of cleaned ingredients', () => {
//       const mockIngredients = [{
//         name: 'Apple',
//         id: 1
//       }]
//       expect(wrapper.state().ingredients).toEqual(mockIngredients)
//     });
//   });
