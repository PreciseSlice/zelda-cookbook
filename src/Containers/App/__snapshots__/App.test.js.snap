// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot 1`] = `
<div
  className="App"
>
  <header
    className="sidebar"
  >
    <div
      id="small-zelda-header"
    >
      <Connect(TypeTabs) />
    </div>
  </header>
  <Connect(SearchBar) />
  <Connect(RecipeContainer) />
</div>
`;
